Install.sh

```diff
@@ -18,6 +18,11 @@ DATE="24.01.21"
 
 echoLogo $VERSION $DATE "Initial Install"
 
+if [ ! -d "/home/$USER" ]; then
+  echo "Home directory for the current user not found. If you are root, switch to a user with a directory under /home/"
+  exit 1
```

Comment:
yes, but if you have like a root server and only use the root user? Having a fallback would still be nice.

Reply:
into /bin then?

Reply:
> into /bin then?

i mean the ffighter application. The problem is, that the check needs to be done in the ffighter Application to, because the scripts will be located in /root/filefighter...

Reply:
idk, should be ok to use /bin